docker/services/ensapi.yml

```
@@ -1,14 +1,13 @@
 services:
   ensapi:
-    container_name: ensapi
     image: ghcr.io/namehash/ensnode/ensapi:${ENSNODE_VERSION:-1.10.1}
     build:
       dockerfile: ./apps/ensapi/Dockerfile
       context: ../..
     ports:
       - "4334:4334"
     healthcheck:
-      test: ["CMD", "curl", "--fail", "-s", "http://localhost:4334/health"]
+      test: [ "CMD", "curl", "--fail", "-s", "http://localhost:4334/health" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)